src/components/Popups/__tests__/PopupReply-test.tsx

```diff
@@ -79,8 +79,13 @@ describe('components/Popups/PopupReply', () => {
             expect(wrapper.exists('[data-testid="ba-Popup-field"]')).toBe(true);
         });
 
+        test.each([true, false])('should disable post button if isEmpty is %s', isEmpty => {
```

Comment:
Typically the inputs are used as the parameters, so `test.each([undefined, 'Test'])`